homeassistant/components/device_tracker/__init__.py

```diff
@@ -58,10 +61,13 @@
 
 CONF_SCAN_INTERVAL = 'interval_seconds'
 DEFAULT_SCAN_INTERVAL = timedelta(seconds=12)
+SCAN_INTERVAL = timedelta(seconds=12)
```

Comment:
Dupe of `DEFAULT_SCAN_INTERVAL` above?

Reply:
`SCAN_INTERVAL` constant in the component has a special meaning since it will be used as default update interval in entity components. Ie it has to be named like this.

Reply:
Does that mean in the future we can remove DEFAULT_SCAN_INTERVAL?

Reply:
Yes!